.rubocop/custom.yml

```diff
@@ -108,7 +108,7 @@ Rails/I18nLazyLookup: # new in 2.14
 Rails/I18nLocaleAssignment: # new in 2.11
   Enabled: true
 Rails/I18nLocaleTexts: # new in 2.14
-  Enabled: true
+  Enabled: false
```

Comment:
In theory, it's convenient to know *exactly* where all your user-facing strings are (the locale file, not a random service or controller or view, etc.), but in practice, I suspect more of the team would expect to find these strings in random spots, given we don't make broad usage of I18n in our apps. Long-winded way of saying: I agree w/you here.